diff --git a/node_modules/@shopify/react-native-skia/src/sksg/HostConfig.ts b/node_modules/@shopify/react-native-skia/src/sksg/HostConfig.ts
index e56dacc..3c7901a 100644
--- a/node_modules/@shopify/react-native-skia/src/sksg/HostConfig.ts
+++ b/node_modules/@shopify/react-native-skia/src/sksg/HostConfig.ts
@@ -177,10 +177,8 @@ export const sksgHostConfig: SkiaHostConfig = {
     _type,
     _oldProps,
     newProps,
-    _updatePayload,
-    _internalInstanceHandle,
-    keepChildren: boolean,
-    _recyclableInstance: null | Instance
+    keepChildren,
+    _newChildSet,
   ) {
     debug("cloneInstance");
     return {
