diff --git a/node_modules/react-native-reanimated/src/createAnimatedComponent/NativeEventsManager.ts b/node_modules/react-native-reanimated/src/createAnimatedComponent/NativeEventsManager.ts
index 1935c56..0b824dd 100644
--- a/node_modules/react-native-reanimated/src/createAnimatedComponent/NativeEventsManager.ts
+++ b/node_modules/react-native-reanimated/src/createAnimatedComponent/NativeEventsManager.ts
@@ -103,7 +103,8 @@ export class NativeEventsManager implements INativeEventsManager {
       // On the first render of a component, we may already receive a resolved view tag.
       return this.#managedComponent.getComponentViewTag();
     }
-    if (componentAnimatedRef.__nativeTag) {
+
+    if (componentAnimatedRef?.__nativeTag) {
       return componentAnimatedRef.__nativeTag ?? -1;
     }
     /*
